npm install fails on Windows

```
λ npm install
npm WARN addRemoteGit Error: Command failed: git -c core.longpaths=true config --get remote.origin.url
npm WARN addRemoteGit
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:202:12)
npm WARN addRemoteGit     at emitTwo (events.js:100:13)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:185:7)
npm WARN addRemoteGit     at maybeClose (internal/child_process.js:850:16)
npm WARN addRemoteGit     at Socket.<anonymous> (internal/child_process.js:323:11)
npm WARN addRemoteGit     at emitOne (events.js:90:13)
npm WARN addRemoteGit     at Socket.emit (events.js:182:7)
npm WARN addRemoteGit     at Pipe._onclose (net.js:479:12)
npm WARN addRemoteGit  git://github.com/bokeh/numbro.git#b4ccab0 resetting remote C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2 because of error: { [Error: Command failed: git -c core.longpaths=true config --get remote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git -c core.longpaths=true config --get remote.origin.url' }
npm ERR! git clone --template=C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/bokeh/numbro.git C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2: Cloning into bare repository 'C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2'...
npm ERR! git clone --template=C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/bokeh/numbro.git C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2: fatal: unable to connect to github.com:
npm ERR! git clone --template=C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/bokeh/numbro.git C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2: github.com[0: 192.30.252.129]: errno=No error
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\bin\\nodejs\\node.exe" "C:\\bin\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v5.9.0
npm ERR! npm  v3.7.3
npm ERR! code 128

npm ERR! Command failed: git -c core.longpaths=true clone --template=C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git://github.com/bokeh/numbro.git C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2
npm ERR! Cloning into bare repository 'C:\Users\dhirschf\AppData\Roaming\npm-cache\_git-remotes\git-github-com-bokeh-numbro-git-b4ccab0-7d900fc2'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.129]: errno=No error
npm ERR!
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\dev\packages\bokeh\bokehjs\npm-debug.log
```

--- a/bokehjs/package.json
+++ b/bokehjs/package.json
@@ -14,7 +14,7 @@
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",
-    "url": "git://github.com/bokeh/bokeh.git"
+    "url": "https://github.com/bokeh/bokeh.git"
   },
   "devDependencies": {
     "browserify": "^11.1.0",
@@ -44,7 +44,7 @@
     "hammerjs": "^2.0.4",
     "mocha": "^2.2.5",
     "mocha-jsdom": "^1.0.0",
-    "numbro": "git://github.com/bokeh/numbro.git#b4ccab0",
+    "numbro": "https://github.com/bokeh/numbro.git#b4ccab0",
     "phantomjs": "1.9.7-15",
     "run-sequence": "^1.0.0",
     "sandboxed-module": "^2.0.0",

